apps/meeting/src/containers/MeetingProviderWrapper/index.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import {
+  MeetingProvider,
+  VoiceFocusProvider,
+} from 'amazon-chime-sdk-component-library-react';
+
+import routes from '../../constants/routes';
+import { NavigationProvider } from '../../providers/NavigationProvider';
+import NoMeetingRedirect from '../NoMeetingRedirect';
+import { Meeting, Home, DeviceSetup } from '../../views';
+import MeetingEventObserver from '../MeetingEventObserver';
+import meetingConfig from '../../meetingConfig';
+import { useAppState } from '../../providers/AppStateProvider';
+
+const MeetingProviderWrapper: React.FC = () => {
+  const { isWebAudioEnabled } = useAppState();
+
+  const meetingConfigValue = {
+    ...meetingConfig,
+    enableWebAudio: isWebAudioEnabled,
+  };
+
+  const getMeetingProviderWrapper = () => {
+    return (
+      <MeetingProvider {...meetingConfigValue}>
+        <NavigationProvider>
+          <Switch>
+            <Route exact path={routes.HOME} component={Home} />
+            <Route path={routes.DEVICE}>
+              <NoMeetingRedirect>
+                <DeviceSetup />
+              </NoMeetingRedirect>
+            </Route>
+            <Route path={routes.MEETING}>
+              <NoMeetingRedirect>
+                <MeetingModeSelector />
+              </NoMeetingRedirect>
+            </Route>
+          </Switch>
+        </NavigationProvider>
+        <MeetingEventObserver />
+      </MeetingProvider>
+    );
+  };
+
+  const getMeetingProviderWrapperWithVF = () => {
+    return (
+      <VoiceFocusProvider>
+        {getMeetingProviderWrapper()}
+      </VoiceFocusProvider>
+    )
+  }
```

Comment:
nit: End this with a semicolon, we end the callback functions with a semicolon (learning from my previous reviews).

Reply:
fixed